### ISSUE TITLE
Upload repos to github

### ISSUE THREAD

**Issue by joelee484** [simplworld/simpl.world.website#63]
After #9, upload the repos to github!

- [x] simpl-authenticator
- [x] simpl-calc-model
- [x] simpl-calc-ui
- [x] simpl-client
- [x] simpl-div-model
- [x] simpl-div-ui
- [x] simpl-docs
- [x] simpl-games-api
- [x] simpl-modelservice
- [x] simpl-react
- [x] simpl-users
- [x] simpl-ui-cookiecutter

**Comment by jefftriplett** [comment]
@joelee484 can I help with any of these? I need `simpl-calc-model` and `simpl-calc-ui` to say that the single player tutorial is complete?

**Comment by frankwiles** [comment]
Reason being it'll probably be a week before I can find the time on those.

**Comment by jefftriplett** [comment]
@frankwiles 👍 wfm. I'll ask you questions tomorrow just to ensure I scrubbed appropriately.

**Comment by jefftriplett** [comment]
@joelee484 @janeeisenstein @frankwiles these are now migrated onto github and I archived them on https://learninglab.githost.io/lldev-team   Is it safe to delete any of these? 

I haven't looked at travis / testing but I assume we want to make that work too? (If so, I'll create a new ticket.)
